site/src/examples/pagination/SiblingCount.tsx

```diff
@@ -0,0 +1,28 @@
+import { ChangeEvent, ReactElement, useState } from "react";
+import { StackLayout } from "@salt-ds/core";
+import { Pagination, Paginator } from "@salt-ds/lab";
+
+export const SiblingCount = (): ReactElement => {
+  const [siblingCount, setSiblingCount] = useState(2);
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const value = parseInt(event.target.value);
+    setSiblingCount(value);
+  };
+
+  return (
+    <>
+      <StackLayout>
+        <input
+          type="number"
+          value={siblingCount?.toString()}
+          onChange={handleChange}
+          style={{ width: 50 }}
+        />
+        <Pagination count={20} page={10}>
+          <Paginator siblingCount={siblingCount} />
```

Comment:
Should the sibling count examples also be interactive? Clicking the arrows doesn't change the number for me

Reply:
fixed